hack/clibyexample/template

```diff
@@ -0,0 +1,12 @@
+{{range .items}}
```

Comment:
`name` or `fullName`? does it look silly to have `oc ` or `oadm ` repeated on every item in the TOC?